templates/Web/Pages/Vue.MasterDetail/src/components/MasterDetailSideBarTab.vue

```diff
@@ -13,8 +13,14 @@
 export default {
   name: "MasterDetailSideBarTab",
   props: {
-    tabText: String,
-    index: Number
+    tabText: {
+      type: String,
+      required: true
+    },
+    index: {
+      type: Number,
+      required: true
+    }
```

Comment:
Out of curiosity - would the validation fail right away if we get some other type rather than `Number` for `index` prop?

Reply:
yes it would, it doesnt crash or stop rendering, but it throws a console.log error